content/pt-br/docs/reference/issues-security/issues.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Rastreador de Issue Kubernetes
+weight: 10
+aliases: [/pt-br/cve/,/pt-br/cves/]
+---
+
+Para reportar um problema de segurança, siga [processo de divulgação de issues do Kubernetes](/docs/reference/issues-security/security/#report-a-vulnerability).
```

Comment:
This sentence is not quite a faithful translation from the original. Perhaps "processo de divulgação de segurança do Kubernetes" would be better suited?